_tests/travis-checks

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+# Execute Markdown Linter
+mdl README.md _posts _pages --style _linter/markdown-linter-style.rb
+
+# Github Pages needs the baseurl field to be "m-lab.github.io" to render properly in forked 
+# repos, but when we deploy to production, we need the baseurl field to be empty. This 
+# command replaces "m-lab.github.io" with an empty string.
+sed -e '/^baseurl/s/\/m-lab\.github\.io//' _config.yml > _config-deploy.yml
+
+# Skips validation checks that take a long time
+arg=""
```

Comment:
OK, that makes sense to me.

That is fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/31ec265f36f15bb9efa7af643d1e90d111da55c8